index.html

```diff
@@ -19,21 +20,171 @@
             </nav>
         </div>
         <section class="headline">
-            <h1>I'm Mohamad. <br> Glad to see you!</h1>
-            <p>I'm a full stack web developer! 
-                I can help you buil website <br> 
-                Look though some of my work and experience! 
-                If you like what you see and you have 
-                a project you need coded, 
-                don't hesitate to contact me. </p>
-            <a class="connect_link" href="">LET'S CONNECT</a>
+            <h1>I'm Alexandra. Glad to see you!</h1>
+            <p>
+                I'm a software developer! I can help you 
+                build a product, feature or website Look 
+                through some of my work and 
+                experience! If you like what you see and
+                have a project you need coded, don't 
+                hesitate to contact me.
+            </p>
+            <h3>
+                <a class="connect_link" href="">LET'S CONNECT</a>
+            </h3>
             <ul class="icons">
                 <li><a href="https://twitter.com/MohametAlmeari"><i class="fa fa-twitter"></i></a></li>
                 <li><a href="https://www.linkedin.com/in/mohamet-almeari"><i class="fa fa-linkedin"></i></a></li>
                 <li><a href="https://github.com/mohametalmeari"><i class="fa fa-github"></i></a></li>
                 <li><a href=""><i class="fa fa-instagram"></i></a></li>
-            </div>
+            </ul>
         </section>
+        
+            <section class="works">
+                <div class="card">
+                    <img class="preview" src="/images/SnapshootPortfolio1.png" alt="Website Preview">
```

Comment:
- [ ] Awesome adding the recommended image, but the image is not displaying probably because your did not reference the ` src `file properly. I tried the suggested line of code below and it worked (there should be a **.** in front of the forward slash) ℹ️. **The same thing is applicable to the three other cards.**

```suggestion
                    <img class="preview" src="./images/SnapshootPortfolio1.png" alt="Website Preview">
```